libs/application/templates/driving-instructor-registrations/src/fields/mock.ts

```diff
@@ -0,0 +1,105 @@
+export const STUDENTS = [
+  {
```

Comment:
same re: including actual users in mock data - and ideally not having any mock data in running code paths (beyond tests)